### PR TITLE
Fixed solo CSS having typographie rules from Foundation

### PR DESCRIPTION
**Pull request by Owlbertz**
This caused issues with other CSS rules.

Addresses #226 

Ping @kball, please check this out.

### CODE REVIEW

File: scss/foundation_dependencies.scss [hunk 1]
```diff
@@ -25,7 +25,6 @@
 
 
 // Imports
-@include foundation-typography;
 @include foundation-visibility-classes;
 
 {{#components}}
```
Comment (kball) [inline scss/foundation_dependencies.scss]: Had the thought looking at this... why are we including so many components? Almost certainly we only need a few...

Comment (Owlbertz) [inline scss/foundation_dependencies.scss]: The components are automatically rendered into the SASS file using the configuration file https://github.com/zurb/joyride/blob/master/gulp/config.js. Only Reveal and Tooltip are used here, plus some utility stuff.

Comment (kball) [inline scss/foundation_dependencies.scss]: 👍

**Comment by Owlbertz** [comment]
I'd go ahead and wrap up a 3.0.1 with this fix then... Seems reasonable enough for me to update for this PR.

**Pull request merged**